src/h3lib/lib/h3Index.c

```diff
@@ -183,7 +183,7 @@ static bool _isValidChildRes(int parentRes, int childRes) {
  * @return int count of maximum number of children (equal for hexagons, less for
  * pentagons
  */
-int H3_EXPORT(maxH3ToChildrenSize)(H3Index h, int childRes) {
+int64_t H3_EXPORT(maxH3ToChildrenSize)(H3Index h, int childRes) {
```

Comment:
Nit: Not sure I understand the full implications, but why `int64_t` here and `uint64_t` elsewhere? This is guaranteed positive as well.

Reply:
Yeah. I'm not sure what the best approach is here with these types. In this case, I've just chosen `int64_t` over `uint64_t` just to match with what we already have with `int64_t H3_EXPORT(numHexagons)(int res)`.